Gemfile.lock

```diff
@@ -2,6 +2,7 @@ PATH
   remote: .
   specs:
     routemaster-drain (2.4.4)
+      addressable
```

Comment:
Do you know why this wasn't added to the list of DEPENDENCIES at the bottom of the file? Seems like it should have been.